kustomize-functions/validate-asm/image/main.go

```diff
@@ -34,6 +34,8 @@ const (
 	containerClusterKind   = "ContainerCluster"
 	containerNodePoolKind  = "ContainerNodePool"
 	apiGroup               = "container.cnrm.cloud.google.com"
+	minimalTotalVCPUs      = 8
```

Comment:
nit: this is normally "minimum", not "minimal"

Reply:
Thanks, done!